app/views/campaigns/new.html.haml

```diff
@@ -0,0 +1,25 @@
+= form_with model: @campaign, local: true do |form|
+  .form-group  
+    .control-label
+      = form.label :title
+    %br/
+    .form-control    
+      = form.text_field :title
+    %br/
+  .form-group  
+    .control-label   
+      = form.label :description
+    %br/
+    .form-control    
+      = form.text_area :description
+    %br/
+  .form-group  
+    .control-label    
+      = form.label :location
+    %br/
+    .form-control    
+      = form.text_field :location
+    %br/
+  .form-group  
+    .form-submit   
+      = form.submit 'Launch Campaign'
```

Comment:
Could you explain why this doesn't work? @tochman I feel like we probably should know this by know, but I really don't have a clue. 
For instance the styling works when we remove the `form.submit`, but obviously there's no way to submit the form then.

Reply:
When you say it does not work, how is that manifested?

Reply:
The submit-button does not exist, so there's nothing to click on.

Reply:
<img width="1439" alt="screen shot 2018-09-04 at 21 21 54" src="https://user-images.githubusercontent.com/33520974/45054370-0aa0fc80-b08d-11e8-9057-6b6267cea49a.png">

Reply:
We made progress though!

Reply:
<img width="825" alt="screen shot 2018-09-04 at 21 55 06" src="https://user-images.githubusercontent.com/33520974/45054416-38864100-b08d-11e8-8889-a8566f61ce7a.png">

Now we have this!